ubuntu_18.04-dev.dockerfile

```diff
@@ -19,13 +26,15 @@ RUN echo "export PATH=$HOME/.local/bin:\$PATH" >> ~/.bashrc \
     && echo "alias python=python3" >> ~/.bashrc \
     && echo "alias nosetests=nosetests3" >> ~/.bashrc
 
+RUN gcc --version
+
 # build MOAB
 RUN cd $HOME/opt \
   && mkdir moab \
   && cd moab \
   && git clone https://bitbucket.org/fathomteam/moab \
   && cd moab \
-  && git checkout -b Version4.9.1 origin/Version4.9.1 \
+  && git checkout -b Version5.0 origin/Version5.0 \
```

Comment:
PyNE dev is won't be compatible with 5.0 until we merge in PyMOAB updates. This should remain as 4.9.1 until then.

Reply:
Furthermore, once we move to v5.0 or v5.1, we will need to also have DAGMC added as a dependency here.